### ISSUE TITLE
Upgrade yarn to version 1.2.10

### ISSUE THREAD

**Issue by altro3** [ScoopInstaller/Main#1894]
Yarn 1.22.10 was published 5 months ago, but it's still version 1.22.5 in the bucket 

https://www.npmjs.com/package/yarn?activeTab=versions

Please, upgrade yarn to latest version.

**Comment by HUMORCE** [comment]
https://classic.yarnpkg.com/

1.22.5 is the stable version.

**Comment by altro3** [comment]
@HUMORCE Hi!

Hm.. very strange. I think this is a bug on the yarn page because yarn 1.X is now frozen. 

But you can look at their official repository in github and npm - latest version 1.22.10. If you try to install yarn through the npm repository, you will have version 1.22.10 installed. 

LInks:

Yarn 1.22.10 Release https://github.com/yarnpkg/yarn/releases/tag/v1.22.10

Npm yarn
![изображение](https://user-images.githubusercontent.com/48474279/110299059-da54c300-8027-11eb-925a-8a2e6c515d55.png)

**Comment by HUMORCE** [comment]
https://github.com/yarnpkg/yarn/releases/tag/v1.22.10

> Please check the changelog for details: https://github.com/yarnpkg/yarn/blob/master/CHANGELOG.md
For technical reasons, **no prebuilt artifacts will be generated for this release**. Please use npm install -g yarn to install it.

**Comment by altro3** [comment]
yeah, this is the last release of yarn 1. Тo more releases are planned. But fix version 1.22.6 is also not included in version 1.22.5.

I understand that the changes are minor in recent releases, but the fact is the fact - the latest version is 1.22.10
